cli/server.go

```diff
@@ -2306,7 +2307,7 @@ func IsLocalhost(host string) bool {
 // future or past migration version.
 //
 // If no error is returned, the database is fully migrated and up to date.
-func ConnectToPostgres(ctx context.Context, logger slog.Logger, driver string, dbURL string, migrate func(db *sql.DB) error) (*sql.DB, error) {
+func ConnectToPostgres(ctx context.Context, logger slog.Logger, driver string, dbURL string, migrate func(db *sql.DB) error, maxConns, idleConns int) (*sql.DB, error) {
```

Comment:
You could probably keep these as int64 and then do the casting down below.

Reply:
`int64` is just a detail of the `serpent` options. I'd rather deal with more realistic types in the rest of the codebase.